testsuite/mockserver/classes.py

```diff
@@ -37,24 +37,39 @@ class SslCertInfo:
 
 @dataclasses.dataclass(frozen=True)
 class MockserverInfo:
-    host: str
-    port: int
     base_url: str
-    ssl: typing.Optional[SslCertInfo]
```

Comment:
why not leave it here? it's possible to run https over unix socket

Reply:
fixed